src/ui/prefs.ui

```diff
@@ -388,6 +388,34 @@ a white jersey	1975	"best cyclist aged 25 or less"</property>
             </child>
           </object>
         </child>
+        <child>
+          <object class="HdyPreferencesGroup">
+            <property name="visible">True</property>
+            <property name="can-focus">False</property>
+            <property name="title" translatable="yes">Advanced Compiler Options</property>
+            <child>
+              <object class="HdyActionRow">
+                <property name="visible">True</property>
+                <property name="can-focus">False</property>
+                <property name="activatable">False</property>
+                <property name="selectable">False</property>
+                <property name="title" translatable="yes">Additional Directory to Search for Extensions</property>
+                <property name="subtitle" translatable="yes">This option is intended only for advanced users of Inform 7. It should have the full path to the parent directory of a directory named "Extensions". The Inform 7 compiler will search this directory for extensions (in addition to searching the usual locations) when compiling all projects. This can be useful when developing extensions, for testing a work-in-progress extension with many different games.</property>
```

Comment:
These were cutting off, so I edited the headline and subtitle a bit to make them shorter, hope that's OK.